data_steward/analytics/cdr_ops/cdr_qa_report_dateshift.py

```diff
@@ -0,0 +1,693 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.7.1
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# + papermill={"duration": 0.023643, "end_time": "2021-02-02T22:30:31.880820", "exception": false, "start_time": "2021-02-02T22:30:31.857177", "status": "completed"} tags=["parameters"]
+project_id = ""
+com_cdr = ""
+new_cdr = ""
+#dataset_id=""
+
+# + papermill={"duration": 0.046903, "end_time": "2021-02-02T22:30:31.927723", "exception": false, "start_time": "2021-02-02T22:30:31.880820", "status": "completed"} tags=["injected-parameters"]
```

Comment:
You can get rid of this cell because it has project information.

Reply:
one quick question, if that cell is removed, how do we know what database or dataset should be used? from the queries i got from QA team, each sheet may use different datasets.

Reply:
You keep the first `parameters` cell, but not the second `injected parameters` cell that is output by papermill. That second cell has passed in parameter values that we do not want to place in the repo. Does that make sense?